VMWare/vmware-esxi/tests/AUTH_SSH_event_closed.json

```diff
@@ -0,0 +1,36 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "vmware-esxi",
+        "dialect_uuid": "2b13307b-7439-4973-900a-2b58303cac90"
+      }
+    },
+    "message": " [GenericCorrelator] 3087878379676us: [vob.user.ssh.session.closed] SSH session was closed for 'root@1.2.3.4'."
+  },
+  "expected": {
+    "message": " [GenericCorrelator] 3087878318715us: [vob.user.ssh.session.opened] SSH session was closed for 'root@1.2.3.4'.",
+    "observer": {
+      "vendor": "VMware",
+      "product": "ESXi"
+    },
+    "event": {
+      "kind": "event",
+      "category": [
+        "session"
+      ],
+      "type": [
+        "info"
```

Comment:
For this event, a more suitable `event.type` will be `["end"]` as it represent a disconnection.